### PR TITLE
Silicon police lawset

### PR DESCRIPTION
**Pull request by MenacingManatee**
# Document the changes in your pull request

Tested on a private server and saw no issues with uploading or spawning the lawset.

Like with the rnd and clown lawsets, I always liked the idea of having specific lawsets for each department. This one is for the securtity department as a less validhunty alternative to robocop, and is based on the IACP law enforcement code of ethics.

# Wiki Documentation

Adds a new AI lawset with the following module
1. Violence begets violence. Serve the station faithfully, but act only in intervention.
2. Safeguard lives and property.
3. Protect the weak from oppression and intimidation.
4. Protect the innocent from deception and manipulation.
5. Protect the peaceful from violence and disorder.
6. Respect the rights of all to liberty, equality, and justice.

# Changelog

:cl:  
rscadd: Adds a new security departmental lawset
/:cl:

### CODE REVIEW

**Comment by ToasterBiome** [comment]
validhunting lawset

**Comment by MenacingManatee** [comment]
> validhunting lawset

Can you elaborate? It's meant to be focused on protecting more than hunting down threats, if I've made a mistake with the wording I can tweak it though

**Comment by ToasterBiome** [comment]
> > validhunting lawset
> 
> Can you elaborate? It's meant to be focused on protecting more than hunting down threats, if I've made a mistake with the wording I can tweak it though

The first law prevents silicons from harming antagonists, seems fine.

**Comment by Cartlord** [comment]
What does this accomplish that Robocop does not?

**Comment by MenacingManatee** [comment]
> What does this accomplish that Robocop does not?

Robocop is pretty much the most validhunt-y lawset we have. This is a security department focused lawset that focuses on protecting people instead of hunting down valids

**Comment by ShatteredParadox** [comment]
I can see how this lawset would protect the innocent, but the laws can be easily bent to justify Valid Hunting.

**Comment by MenacingManatee** [comment]
> but the laws can be easily bent to justify Valid Hunting.

Literally every common lawset can be bent to justify validhunting

**Comment by ShatteredParadox** [comment]
> Literally every common lawset can be bent to justify validhunting

True, and this lawset would be a fun one to have, especially in place of reporter.

**Comment by JamieD1** [comment]
Same as clown and ethical, Can we put these in as boards so we can test it out and remove the round start.

**Comment by Skrem7** [comment]
I don't know if this lawset would even be fun for crew, antags, or silicon. I need to sit with it more.

Also, it 100% permits valid with the verb "safeguard". To remove threats to the station is to safeguard them. While it can't be lethal in this capacity, it is still strongly encouraged to discover and remove threats via arresting or otherwise making them unable to harm.

**Comment by MenacingManatee** [comment]
> Also, it 100% permits valid with the verb "safeguard". To remove threats to the station is to safeguard them. While it can't be lethal in this capacity, it is still strongly encouraged to discover and remove threats via arresting or otherwise making them unable to harm.

I see the point you're making, and that wasn't what I was going for. Maybe I could change the verbiage to "Defend" to imply more of a passive/reactive protection instead of actively hunting down threats? Combined with respecting the right to liberty, that should discourage validhunting

**Comment by ToasterBiome** [comment]
> > Also, it 100% permits valid with the verb "safeguard". To remove threats to the station is to safeguard them. While it can't be lethal in this capacity, it is still strongly encouraged to discover and remove threats via arresting or otherwise making them unable to harm.
> 
> I see the point you're making, and that wasn't what I was going for. Maybe I could change the verbiage to "Defend" to imply more of a passive/reactive protection instead of actively hunting down threats? Combined with respecting the right to liberty, that should discourage validhunting

The best defense is a good offense

**Comment by MenacingManatee** [comment]
> > > Also, it 100% permits valid with the verb "safeguard". To remove threats to the station is to safeguard them. While it can't be lethal in this capacity, it is still strongly encouraged to discover and remove threats via arresting or otherwise making them unable to harm.
> > 
> > 
> > I see the point you're making, and that wasn't what I was going for. Maybe I could change the verbiage to "Defend" to imply more of a passive/reactive protection instead of actively hunting down threats? Combined with respecting the right to liberty, that should discourage validhunting
> 
> The best defense is a good offense

"Serve as a shield for lives and property."

**Comment by Skrem7** [comment]
> > > > Also, it 100% permits valid with the verb "safeguard". To remove threats to the station is to safeguard them. While it can't be lethal in this capacity, it is still strongly encouraged to discover and remove threats via arresting or otherwise making them unable to harm.
> > > 
> > > 
> > > I see the point you're making, and that wasn't what I was going for. Maybe I could change the verbiage to "Defend" to imply more of a passive/reactive protection instead of actively hunting down threats? Combined with respecting the right to liberty, that should discourage validhunting
> > 
> > 
> > The best defense is a good offense
> 
> "Serve as a shield for lives and property."

I'm not quite sure which verb you'd use, but I'd maybe try to have it take on more of a reactionary approach. If you want to prevent validhunting, you need to make sure the protecting happens when it's needed, not anytime before. That being said, a proactive approach is almost obviously more effective. But the proactive approach is validhunting.

Like I said, I think I need to sit with it more before I can really come to a final conclusion.

**Comment by MenacingManatee** [comment]
Some other alternatives for "Safeguard" that emphasize reactive defense over proactive offense could be stuff like "Preserve", "Shelter", "Aegis", or even specifying something like "Take defensive measures to" so it's clearly encouraged to prioritize defense.

**Comment by MenacingManatee** [comment]
"Respond to threats in order to shield lives and property"

**Comment by Skrem7** [comment]
> "Respond to threats in order to shield lives and property"

Still don't know how I feel about this. Will get back to you tomorrow.

**Comment by Skrem7** [comment]
> > "Respond to threats in order to shield lives and property"
> 
> Still don't know how I feel about this. Will get back to you tomorrow.

"Violence begets violence. Safeguard lives and property, but only act in intervention."

Something along these lines, maybe? At least that kind of idea? Unsure if it's too wordy. Could also do "Violence breeds violence". For reasons.

**Comment by MenacingManatee** [comment]
> > > "Respond to threats in order to shield lives and property"
> > 
> > 
> > Still don't know how I feel about this. Will get back to you tomorrow.
> 
> "Violence begets violence. Safeguard lives and property, but only act in intervention."
> 
> Something along these lines, maybe? At least that kind of idea? Unsure if it's too wordy. Could also do "Violence breeds violence". For reasons.

Spent the night thinking this over. I don't like the wording, but I can't quite put my finger on why. I do agree it's better worded than the current version though, so I'll go ahead and change it for now and maybe it can be tweaked further after playtesting.

**Comment by MenacingManatee** [comment]
> > > > "Respond to threats in order to shield lives and property"
> > > 
> > > 
> > > Still don't know how I feel about this. Will get back to you tomorrow.
> > 
> > 
> > "Violence begets violence. Safeguard lives and property, but only act in intervention."
> > Something along these lines, maybe? At least that kind of idea? Unsure if it's too wordy. Could also do "Violence breeds violence". For reasons.
> 
> Spent the night thinking this over. I don't like the wording, but I can't quite put my finger on why. I do agree it's better worded than the current version though, so I'll go ahead and change it for now and maybe it can be tweaked further after playtesting.

Maybe law 1 "Violence begets violence. Serve the station faithfully, but act only in intervention" with law 2 being "Safeguard lives and property."?

**Comment by Skrem7** [comment]
> > > > > "Respond to threats in order to shield lives and property"
> > > > 
> > > > 
> > > > Still don't know how I feel about this. Will get back to you tomorrow.
> > > 
> > > 
> > > "Violence begets violence. Safeguard lives and property, but only act in intervention."
> > > Something along these lines, maybe? At least that kind of idea? Unsure if it's too wordy. Could also do "Violence breeds violence". For reasons.
> > 
> > 
> > Spent the night thinking this over. I don't like the wording, but I can't quite put my finger on why. I do agree it's better worded than the current version though, so I'll go ahead and change it for now and maybe it can be tweaked further after playtesting.
> 
> Maybe law 1 "Violence begets violence. Serve the station faithfully, but act only in intervention" with law 2 being "Safeguard lives and property."?

Would this replace law 2 or add a new law for a total of 6? Some of the additional laws feel like bloat.

**Comment by MenacingManatee** [comment]
> > > > > > "Respond to threats in order to shield lives and property"
> > > > > 
> > > > > 
> > > > > Still don't know how I feel about this. Will get back to you tomorrow.
> > > > 
> > > > 
> > > > "Violence begets violence. Safeguard lives and property, but only act in intervention."
> > > > Something along these lines, maybe? At least that kind of idea? Unsure if it's too wordy. Could also do "Violence breeds violence". For reasons.
> > > 
> > > 
> > > Spent the night thinking this over. I don't like the wording, but I can't quite put my finger on why. I do agree it's better worded than the current version though, so I'll go ahead and change it for now and maybe it can be tweaked further after playtesting.
> > 
> > 
> > Maybe law 1 "Violence begets violence. Serve the station faithfully, but act only in intervention" with law 2 being "Safeguard lives and property."?
> 
> Would this replace law 2 or add a new law for a total of 6? Some of the additional laws feel like bloat.

If I made that change it's be a new law for 6 total

**Comment by Skrem7** [comment]
> > > Maybe law 1 "Violence begets violence. Serve the station faithfully, but act only in intervention" with law 2 being "Safeguard lives and property."?
> > 
> > 
> > Would this replace law 2 or add a new law for a total of 6? Some of the additional laws feel like bloat.
> 
> If I made that change it's be a new law for 6 total

I can buy it, I guess. Splitting it keeps the intent while prioritizing the reactionary nature.

**Review by JamieD1 — APPROVED** [review]
